compiler/bcq-tools/generate_bcq_metadata.py

```diff
@@ -21,6 +21,9 @@
 import argparse
 import sys
 
+# TODO Find better way to suppress trackback on error
+sys.tracebacklimit = 0
+
```

Comment:
please do not mix compile/* module

Reply:
@seanshpark As I understand, you mean to exclude `compiler/*` from this PR, and make separate PR. I will do. If you mean different thing, please let me know. Thank you for review.

Reply:
This PR title starts with `[one-cmds]`, that means include `compile/one-cmds/*` but exclude others.

Reply:
You're right. Thank you for correcting me. I've just updated.